service/templates/admin.html

```diff
@@ -604,6 +605,14 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 										<label>Host Allocator</label>
 										<input type="text" ng-model="Settings.scheduler.host_allocator">
 									</md-input-container>
+									<md-input-container class="control" style="width:60%;">
```

Comment:
Adding this in the middle messed up the alignment of the controls below it. I think you either need to alternate the styling on everything below, or add it to the end

Reply:
I can just add it to the end.

Reply:
Actually I think I added an extra div tag, I think I just needed to get rid of it.